es/index.js

```diff
@@ -36,9 +36,11 @@ export default function axiosRetry(axios, { retries = 3 } = {}) {
       return Promise.reject(error);
     }
 
+    const maxRetries = config.retries || retries;
```

Comment:
Instead of `maxRetries` I'd simply use `config.retries` if it's set, using the global `retries` option as a default value.

Thanks for the PR by the way ;)